reconcile/utils/terraform_resource_spec.py

```diff
@@ -1,6 +1,8 @@
 from dataclasses import dataclass, field
 import json
 from typing import Any, Optional, cast
+import base64
+from reconcile.utils.openshift_resource import OpenshiftResource as OR
```

Comment:
Can we stop the pattern of calling `OpenshiftResource` "`OR`"? Short identifiers should be used in local variables only.

Reply:
done